docs/metrics.md

```diff
@@ -159,4 +159,8 @@ An event fired when user goes over the pad limit (15000 character)
 - `cd3`
 - `cd4`
 - `cd5`
-- `cd6`
\ No newline at end of file
+- `cd6`
+
+#### `idb-fail`
+
```

Comment:
You'd know more than I do, but are any event contexts needed for this metric?

Reply:
Ah yes, I can add some 👍